### PR TITLE
fixed post

### PR DESCRIPTION
**Pull request by dogeared**
fix stuff and junk

### CODE REVIEW

File: _source/_posts/2020-02-07-transactional-mfa.md [hunk 1]
```diff
@@ -40,31 +40,31 @@ Did you know that outside of the standard authentication scenarios, a developer
...
+## Create a Custom SAML Application in Okta
 
-Classic UI Console:
+In this example, you will be using a new early-access feature called [Step-up authentication with Okta Session](https://developer.okta.com/docs/reference/api/authn/#step-up-authentication-with-okta-session). This is a new feature that allows you to get a `stateToken` from Okta and use it as a parameter within the Okta Sign-In widget so that you can bootstrap the Okta Sign-In widget to do MFA automatically. If you don't have this feature enabled, I would suggest creating a support ticket to have this feature enabled on your Okta tenant.
 
-{% img blog/transactional-mfa/classic-console.png alt:"Classic Console" width:"800" %}{: .center-image }
+To do this, you will need to create a custom SAML application.
 
-Click **Applications** -> **Add Application**. Next, click **Add Application** -> **Create New App** -> **Select Web and SAML**.
+Click **Applications** > **Applications**. Next, click **Add Application** > **Create New App** > **SAML 2.0**. Click **Create**.
```
Comment (mraible) [inline _source/_posts/2020-02-07-transactional-mfa.md]: This seems wrong. Double Applications?

Comment (dogeared) [reply _source/_posts/2020-02-07-transactional-mfa.md]: yeah... It's how the developer ui is

Comment (dogeared) [reply _source/_posts/2020-02-07-transactional-mfa.md]: I meant the classic ui

Comment (dogeared) [reply _source/_posts/2020-02-07-transactional-mfa.md]: image

**Pull request merged**